### PR TITLE
ENH: Allow updating GBQ table schemas

### PR DESCRIPTION
**Pull request by joshowen**
- [x] tests added / passed
 - [x] passes ``git diff upstream/master | flake8 --diff``
 - [x] whatsnew entry

### CODE REVIEW

**Comment by joshowen** [comment]
Looks like the 1 failing test was because of a conda repo issue.

**Comment by jreback** [comment]
what is the purpose of this? is there an issue for this change?

**Comment by joshowen** [comment]
There isn't a tracked issue.  I was running into an issue where I wanted to append to existing GBQ tables but either had a new column or were missing a few legacy columns and that caused the append to fail.

File: doc/source/whatsnew/v0.19.2.txt [hunk 1]
```diff
@@ -64,3 +64,6 @@ Bug Fixes
 
 
 - Explicit check in ``to_stata`` and ``StataWriter`` for out-of-range values when writing doubles (:issue:`14618`)
```
Comment (jreback) [inline doc/source/whatsnew/v0.19.2.txt]: move to 0.20.0

**Comment by jreback** [comment]
cc @parthea thoughts?

**Review by parthea — COMMENTED** [review]
Cool feature! I have a few minor observations.

**Comment by jreback** [comment]
status of this?

**Comment by joshowen** [comment]
I haven't had a chance to make the changes.  I'll try to get them done this week.

**Comment by jreback** [comment]
@joshowen this has migrated to https://github.com/pydata/pandas-gbq, feel free to open issue / PR there.

**Pull request closed without merging**